posthog/client.py

```diff
@@ -137,7 +139,11 @@ def validate_client_query_id(
     if not client_query_id:
         return None
     if not client_query_team_id and (not args or "team_id" not in args):
-        raise Exception("Query needs to have a team_id arg if you've passed client_query_id")
+        if DEBUG or TEST:
+            raise Exception("Query needs to have a team_id arg if you've passed client_query_id")
```

Comment:
lacking context here, is it possible to pass in a `client_query_id` but not `team_id` in production?

Reply:
Grabbing the team_id from the args was the wrong solution. Now you just need to set it explicitly